spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'spec_helper'
+
+RSpec.describe FacilityFactory do
+  describe '#DMV facility data' do
+    it 'creates facilities' do
+      facility = FacilityFactory.new
+      colorado_facilities = DmvDataService.new.co_dmv_office_locations
+
+      expect(facility.create_facility(colorado_facilities)[0]).to be_instance_of Facility
+      facility.create_facility(colorado_facilities).each do |data|
+      expect(data).to be_instance_of Facility
+      end
```

Comment:
By testing every element in the return value of `facility.create_facility(colorado_facilities)`, you have eliminated the need for line 9